packages/shell-api/src/database.ts

```diff
@@ -809,19 +809,50 @@ export default class Database extends ShellApiWithMongoClass {
     );
   }
 
+  async _getCurrentOperations(opts: Document | boolean) {
+    const legacyCurrentOpOptions = typeof opts === 'boolean'
+      ? ({ '$all': opts, '$ownOps': false })
+      : ({ '$all': !!opts.$all, '$ownOps': !!opts.$ownOps });
+
+    const pipeline: Document[] = [{
+      $currentOp: {
+        'allUsers': !legacyCurrentOpOptions.$ownOps,
+        'idleConnections': legacyCurrentOpOptions.$all,
+        'truncateOps': false,
+      }
+    }];
+
+    if (typeof opts === 'object') {
+      const matchingFilters: Document = {};
+      for (const filtername in opts) {
+        if (Object.prototype.hasOwnProperty.call(opts, filtername)) {
+          if (filtername !== '$ownOps' && filtername !== '$all' && filtername !== '$truncateOps') {
```

Comment:
Same goes for excluding `$truncateOps` here - was pulled from old shell's implementation